src/features/governance/types.ts

```diff
@@ -59,11 +59,13 @@ export enum ProposalStage {
   Executed = 6,
   Withdrawn = 7,
   Rejected = 8,
+  // Used for Proposals that have zero transactions and therefore execution would only be symbolic.
+  // We still want to show that the proposal was successful.
+  Adopted = 9,
```

Comment:
other words like Passed were considered. Ultimately Passed is to generic, Approved is already used for other concepts in governance. Ratified should be for treaties. 

Adopted in legal speak means a resolution has gone thru all stages and is ready to be implemented.